rotkehlchen/chain/ethereum/modules/aave/v2/decoder.py

```diff
@@ -276,5 +325,10 @@ def addresses_to_decoders(self) -> dict[ChecksumEvmAddress, tuple[Any, ...]]:
             string_to_evm_address('0x7d2768dE32b0b80b7a3454c06BdAc94A69DDc7A9'): (self._decode_lending_pool_events,),  # AAVE_V2_LENDING_POOL  # noqa: E501
         }
 
+    def enricher_rules(self) -> list[Callable]:
```

Comment:
Won't it be possible to handle decoding when a specific event is emitted instead of having an enrichment rule?

Reply:
Liquidations happen in a transaction started by other user and can be against a different contract, like a bot, so it doesn't need to interact directly with the aave proxy.